### ISSUE TITLE
Have to update CSS

### ISSUE THREAD

**Issue by championshuttler** [championshuttler/easyFill#16]
Need a new CSS simple design without using bootstrap.

**Comment by joncancode** [comment]
Hi, I'd love to help. Can I have a link to the file(s)?

**Comment by raviigarg** [comment]
Can i help?

**Comment by championshuttler** [comment]
Yeah @joncancode you can work and @garg-ravi24  you too :)

**Comment by tjeufoolen** [comment]
Still any help wanted?

**Comment by championshuttler** [comment]
@Athyk yes i am planning to remove jquery file and write core js only what we need as there is so much code in jquery but we dont need it so you can work on it if you are comfortable with it

**Comment by tjeufoolen** [comment]
I shall take a look at it! :)

**Comment by tjeufoolen** [comment]
#18 done.
